src/codeu/chat/util/Scheduler.java

```diff
@@ -0,0 +1,26 @@
+package codeu.chat.util;
+
+import java.util.PriorityQueue;
+
+// SCHEDULER
+//
+// Given scheduled events, the Scheduler will check whehter a timer has expired and run its handler
+// if necessary.
+public class Scheduler {
+  private final PriorityQueue<Scheduled> scheduled = new PriorityQueue<>();
```

Comment:
`scheduled` as a variable name is confusing when there's a class called Scheduled. Change to `events`, or at least pluralize it.